modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators validating the data
+
+## Scope
+
+The Candidate Parameters modules stores all information 
+for a candidate that are not timepoint dependent. All data 
+being collect during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Any one of them
+is sufficient to have access to the module.
+
+candidate_parameter_view
+    - This permission gives the user viewing access to the module and all its tabs.
+
+candidate_parameter_edit
```

Comment:
would tend to have those two permissions displayed as a list:

- candidate_parameter_view - This permission gives the...
- candidate_parameter_edit - This permission gives the...